srcpkgs/pcb/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'pcb'
 pkgname=pcb
-version=20140316
-revision=4
+version=4.1.2
```

Comment:
That version change won't work because `20140316` > `4.1.2`. This is why packages with a date stamp should use `0.0.<date>`. Now we can only try to use `reverts=20140316_4`.